ci/run_pytests.sh

```diff
@@ -26,6 +26,7 @@ cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/tests/
 LEGATE_TEST=${LEGATE_TEST:-1} \
 legate \
     --gpus "$(nvidia-smi -L | wc -l)"\
+    --fbmem=4000 \
```

Comment:
Somehow, it seems legate is reserving almost all available GPU memory now, and that just doesn't work for us...

Reply:
@magnatelee does that sound right to you?

Reply:
yeah like @manopapad said on slack, the default behavior now tries to reserve the entire available GPU memory. we'll be transitioning to a mode where things are allocated incrementally.